index.bs

```diff
@@ -359,36 +415,518 @@ To <dfn>sanitizeToString</dfn> a given |input|, run these steps:
+          "longdesc": ["*"],
+          "loop": ["*"],
+          "low": ["*"],
+          "lowsrc": ["*"],
+          "manifest": ["*"],
+          "marginheight": ["*"],
+          "marginwidth": ["*"],
+          "max": ["*"],
+          "maxlength": ["*"],
+          "mayscript": ["*"],
+          "media": ["*"],
+          "method": ["*"],
+          "min": ["*"],
+          "minlength": ["*"],
+          "multiple": ["*"],
+          "muted": ["*"],
+          "name": ["*"],
+          "nohref": ["*"],
+          "nomodule": ["*"],
+          "nonce": ["*"],
+          "noresize": ["*"],
+          "noshade": ["*"],
+          "novalidate": ["*"],
+          "nowrap": ["*"],
+          "object": ["*"],
+          "open": ["*"],
+          "optimum": ["*"],
+          "part": ["*"],
+          "pattern": ["*"],
+          "placeholder": ["*"],
+          "playsinline": ["*"],
+          "ping": ["*"],
+          "policy": ["*"],
+          "poster": ["*"],
+          "preload": ["*"],
+          "pseudo": ["*"],
+          "readonly": ["*"],
+          "referrerpolicy": ["*"],
+          "rel": ["*"],
+          "reportingorigin": ["*"],
+          "required": ["*"],
+          "resources": ["*"],
+          "rev": ["*"],
+          "reversed": ["*"],
+          "role": ["*"],
+          "rows": ["*"],
+          "rowspan": ["*"],
+          "rules": ["*"],
+          "sandbox": ["*"],
+          "scheme": ["*"],
+          "scope": ["*"],
+          "scrollamount": ["*"],
+          "scrolldelay": ["*"],
+          "scrolling": ["*"],
+          "select": ["*"],
+          "selected": ["*"],
+          "shadowroot": ["*"],
+          "shadowrootdelegatesfocus": ["*"],
+          "shape": ["*"],
+          "size": ["*"],
+          "sizes": ["*"],
+          "slot": ["*"],
+          "span": ["*"],
+          "spellcheck": ["*"],
+          "src": ["*"],
+          "srcset": ["*"],
+          "srcdoc": ["*"],
+          "srclang": ["*"],
+          "standby": ["*"],
+          "start": ["*"],
+          "step": ["*"],
+          "style": ["*"],
+          "summary": ["*"],
+          "tabindex": ["*"],
+          "target": ["*"],
+          "text": ["*"],
+          "title": ["*"],
+          "topmargin": ["*"],
+          "translate": ["*"],
+          "truespeed": ["*"],
+          "trusttoken": ["*"],
+          "type": ["*"],
+          "usemap": ["*"],
+          "valign": ["*"],
+          "value": ["*"],
+          "valuetype": ["*"],
+          "version": ["*"],
+          "vlink": ["*"],
+          "vspace": ["*"],
+          "virtualkeyboardpolicy": ["*"],
+          "webkitdirectory": ["*"],
+          "width": ["*"],
+          "wrap": ["*"]
+    }
+    ```
+
+: Default Configuration
+
+:: The <dfn>default configuration</dfn> has the following value:
+    ```
+    {
+      "allowUnknownElements": false
+      "allowUnknownAttributes": false,
+      "allowCustomElements": false,
+      "allowElements": [
```

Comment:
Same nit as above, plus I'd prefer we figure out a way to name it once in the spec and then reference it from there instead of having it fully enumerated in two places.

Reply:
Not sure I agree. The baseline and the default config will differ in subtle ways. Otherwise, we could just merge them again.

Arguably, it'd make sense to have an easier way to construct them, like:
default = { allowElement: baseline.allowElements + ["bla", "bla", "blubb"] }
Or so. That'd likely be much easier to understand.

Incidentally, today someone brought up exactly that point when discussing allowUnknownElements. So if someone wants to add an "unknown" element to the list, they shouldn't have to research and repeat the entire allow-list. I guess we're missing a mechanism there.

(Could be: Default config exposed, so people can easily build on it. Or every Sanitizer exposes their config. Or "inheritance", where you name a "parent" Sanitizer. Or... )

Reply:
Would it make sense to add a (non-normative, I guess?) section that explains how the values were constructed? Ideally in executable form.

Something like: https://github.com/otherdaniel/purification/blob/default-construction/resources/configurations.html

The idea is that one can copy&paste the results into the spec (or generate them as part of the deploy script). If all goes well, the code version could be more easily readable & modifyable than the spec version, but we'd still have a definite version in the spec.